WooCommerce/Classes/Authentication/Prologue/LoginPrologueViewController.swift

```diff
@@ -106,17 +109,19 @@ private extension LoginPrologueViewController {
 
         addChild(carousel)
         view.addSubview(carousel.view)
-        if isNewToWooCommerceButtonShown {
-            NSLayoutConstraint.activate([
-                carousel.view.leadingAnchor.constraint(equalTo: view.leadingAnchor),
-                carousel.view.trailingAnchor.constraint(equalTo: view.trailingAnchor),
-                carousel.view.topAnchor.constraint(equalTo: view.topAnchor),
-                carousel.view.bottomAnchor.constraint(equalTo: newToWooCommerceButton.topAnchor,
-                                                      constant: -Constants.spacingBetweenCarouselAndNewToWooCommerceButton),
-            ])
-        } else {
-            view.pinSubviewToAllEdges(carousel.view)
-        }
+        let bottomConstraint: NSLayoutConstraint = {
+            guard isNewToWooCommerceButtonShown else {
+                return carousel.view.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+            }
+            return carousel.view.bottomAnchor.constraint(equalTo: newToWooCommerceButton.topAnchor,
+                                                         constant: -Constants.spacingBetweenCarouselAndNewToWooCommerceButton)
```

Comment:
maybe we can remove `isNewToWooCommerceButtonShown` (`newToWooCommerceLinkInLoginPrologue` feature flag), I don't think we plan to bring it back (I'll post a question in the team channel to confirm later)

Reply:
I think we should remove this in a separate PR after getting confirmation from the team. I'll merge this PR for now.